### PR TITLE
fix(properties-panel): render validation error below checkbox labes

### LINKED ISSUE #359: Validation message in checkbox component moves checkbox label
__Describe the Bug__

When using `validate` for an `entryFactory.checkbox` component, the rendered validation message will move the label of the checkbox.
Peek 2020-09-14 11-56

__Steps to Reproduce__

1. Add a `validate` function to a checkbox generated using `entryFactory.checkbox(...)`.
2. Make the validation strike

__Expected Behavior__

The validation message appears below the checkbox.

__Environment__

Observed in:
* Zeebe modeler 0.10.0
*  bpmn-js-properties-panel 0.35.0

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

Closes #359

Also added testcases for relevant html validation rendering

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: test/spec/PropertiesPanelSpec.js [hunk 1]
```diff
@@ -505,4 +508,141 @@ describe('properties-panel', function() {
...
+
+  describe('integration test', function() {
+
+    describe('validation', function() {
+
+      function alwaysValidateFunc() {
+        return 'Validation Message';
+      }
+
+      it('should render for autoSuggestTextBox', inject(function(propertiesPanel) {
```
[the remark this answers is not in the document]
Comment (MaxTru) [reply test/spec/PropertiesPanelSpec.js:520]: My understanding is that this integration test tests `propertiesPanel.applyValidationErrors`, hence I added it to `PropertiesPanelSpec`. Intuition behind that: if I would not have used css to fix this bug, I would have adjusted `propertiesPanel.applyValidationErrors`. Now in that case, these tests would have helped to ensure that the fix does not break anything.

[the remark this answers is not in the document]
Comment (pinussilvestrus) [reply test/spec/PropertiesPanelSpec.js:520]: Got it, thanks for the clarification 👍

File: styles/groups.less [hunk 2]
```diff
@@ -24,6 +24,18 @@
     }
   }
 
+  > .bpp-checkbox {
```
Comment (pinussilvestrus) [inline styles/groups.less:27]: I think we should add this to the `properties.less`, since it's more properties than groups related.

**Review by barmac — APPROVED** [review]

File: styles/groups.less (hunk 2 above)
Comment (MaxTru) [reply styles/groups.less]: Done

**Pull request merged**